policy/modules/admin/netutils.te

```diff
@@ -142,6 +142,7 @@ logging_send_syslog_msg(ping_t)
 miscfiles_read_localization(ping_t)
 
 userdom_use_inherited_user_terminals(ping_t)
+term_use_unallocated_ttys(ping_t)
```

Comment:
What is the use case for this?

Reply:
Booting in recovery mode (boot failed enter root password) and running ping.